assignment/src/com/assignmentweek2/person/Doctor.java

```diff
@@ -0,0 +1,41 @@
+package com.assignmentweek2.person;
+
+public class Doctor extends Person{
+    protected String name;
+    protected String address;
```

Comment:
Use private instead of protected as we are not using these variables outside of this class.

Reply:
changed